py/apps/tour/Dockerfile-runtime

```diff
@@ -0,0 +1,41 @@
+FROM cgr.dev/chainguard/wolfi-base:latest@sha256:91ed94ec4e72368a9b5113f2ffb1d8e783a91db489011a89d9fad3e3816a75ba
```

Comment:
Would be better to name it `*.dockerfile` to get proper syntax highlighting etc.